glad/lang/c/loader/gl.py

```diff
@@ -43,7 +43,8 @@
         num_exts_i = 0;
         glGetIntegerv(GL_NUM_EXTENSIONS, &num_exts_i);
         if (num_exts_i > 0) {
-            exts_i = (char **)realloc((void *)exts_i, (size_t)num_exts_i * (sizeof *exts_i));
+            char **tmp_exts_i = (char **)realloc((void *)exts_i, (size_t)num_exts_i * (sizeof *exts_i));
+            exts_i = tmp_exts_i;
```

Comment:
This is a workaround for the problem but doesn't seem to solve it, that if the reallocation fails the original memory is leaked. I think we would have to free the original memory then return with an error code.